README.md

```diff
@@ -193,6 +193,7 @@ include:
 | `CONFIG_DONGLE_SCREEN_OUTPUT_ACTIVE`                           | bool | y       | If the Output Widget should be active or not.                                                                                                                    |
 | `CONFIG_DONGLE_SCREEN_BATTERY_ACTIVE`                          | bool | y       | If the Battery Widget should be active or not.                                                                                                                   |
 | `CONFIG_DONGLE_SCREEN_AMBIENT_LIGHT_TEST`                      | bool | n       | If enabled, the ambient light sensor will be mocked to adjust screen brightness.                                                                                 |
+| `CONFIG_DONGLE_SCREEN_SYSTEM_ICON`                             | int  | 0       | The icon to display when the 'LGUI'/'RGUI' is pressed. (0: macOS, 1: Linux, 2: Windows)                                                                          |
```

Comment:
We could place this option more prominent in the options list as I believe this might be interesting for many to change. 
Currently the list not not sorted alphabetically.

Reply:
Moved to the third place to not separat any other options. Kept the screen options at the top as they are the ones people want to find fast if it appears wrong after startup.